src/coreclr/jit/emitwasm.cpp

```diff
@@ -479,6 +582,34 @@ void emitter::emitDispIns(
         }
         break;
 
+        case IF_LOCAL_DECL:
+        {
+            cnsval_ssize_t count   = emitGetLclVarDeclCount(id);
+            WasmValueType  valType = emitGetLclVarDeclType(id);
+            int            offs    = id->idDebugOnlyInfo()->lclOffset;
```

Comment:
I would move this down to the `if (m_debugInfoSize > 0)` branch. Right now it is going to access nominally uninitialized memory with `m_debugInfoSize == 0`.